src/Umbraco.Core/Runtime/CoreRuntime.cs

```diff
@@ -182,7 +180,7 @@ protected virtual IFactory Boot(IRegister register, DisposableTimer timer)
                 AcquireMainDom(MainDom);
 
                 // determine our runtime level
-                DetermineRuntimeLevel(databaseFactory, ProfilingLogger);
+                //DetermineRuntimeLevel(databaseFactory, ProfilingLogger);
```

Comment:
Had to disable the DetermineRuntimeLevel here since the SQL CE database might not exists yet, causing an exception.